tjreports/site/views/reports/tmpl/default.php

```diff
@@ -184,23 +159,75 @@ class='icon-download'></span>" . JText::_('COM_TJREPORTS_CSV_EXPORT') . "</a>";
 							</ul>
 						</div>
 					</div>
+				<div class="span2">
+					<?php
+						if(!empty($reportId)):
+						echo "<a class='btn' class='button'
+						type='submit'  href='?option=com_tjreports&task=reports.csvexport'><span title='Export'
+						class='icon-download'></span>" . JText::_('COM_TJREPORTS_CSV_EXPORT') . "</a>";
+						endif
+					?>
+				</div>
+				<div class="span2">
+					<div id="reportPagination" class="pull-right ">
+						<select id="list_limit" name="list[limit]" class="input-mini chzn-done" onchange="getFilterdata(0, '','paginationLimit')">
```

Comment:
@hemantpurohit1 : use model pagination method. It will not work as 20 will always be selected.